modules/analytics/src/views/full/ItemsList.tsx

```diff
@@ -28,7 +28,7 @@ const ItemsList: FC<Props> = props => {
       <ol>
         {items.map((item, index) => (
           <li key={index}>
-            <a onClick={item.onClick} className={!item.onClick && style.disabled}>
+            <a onClick={item.onClick} className={cx({ [style.disabled]: !!item.onClick })}>
```

Comment:
I misread the condition, it should be the opposite. 
![image](https://user-images.githubusercontent.com/42552874/110703325-a3b2b000-81c1-11eb-9a12-54baf16ee07c.png)

Reply:
I think Yann is right, it should be `[style.disabled]: !item.onClick`

Reply:
@EFF @allardy fixed